clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/converters/servergroup/CloneKubernetesAtomicOperationConverter.groovy

```diff
@@ -33,6 +34,9 @@ class CloneKubernetesAtomicOperationConverter extends AbstractAtomicOperationsCr
   }
 
   CloneKubernetesAtomicOperationDescription convertDescription(Map input) {
-    KubernetesAtomicOperationConverterHelper.convertDescription(input, this, CloneKubernetesAtomicOperationDescription)
+    def converted = KubernetesAtomicOperationConverterHelper.convertDescription(input, this, CloneKubernetesAtomicOperationDescription)
+    converted.sourceCredentials = (KubernetesNamedAccountCredentials) this.getCredentialsObject(converted.source.account)
```

Comment:
@lwander are you referring to `KubernetesAtomicOperationConverterHelper.convertDescription`? I handled it here because it's specific to the Clone description. I can move it, what did you have in mind?